tests/unit/views/services/camera_test.js

```diff
@@ -0,0 +1,112 @@
+import Rx from 'rxjs';
+import React from 'components/react';
+
+import CameraServiceView from 'js/views/services/camera';
+
+// See capabilities at https://facebook.github.io/react/docs/test-utils.html
+const TestUtils = React.addons.TestUtils;
+
+function waitFor(fn) {
```

Comment:
This is an experiment, not sure if you like it :) Here I just want to play with the way to test async actions where we don't have access to the underlying promises to wait on (e.g a bunch of async actions in DOM event handler). Idea is exactly the same as ```waitFor``` from MarionetteJS and partially borrowed from [1].

Tell me what you think!

P.S. Such utility methods should be moved to a separate file, but I like the rule - ```Do it for yourself, if you/someone-else need(s) in another file - only then make it generic``` :)

[1] http://gregbabiars.com/acceptance-testing-a-react-app-using-react-test-utils-pretender-and-rxjs/